synchronizer/models.py

```diff
@@ -498,6 +499,26 @@ def delete(cls, sync_id):
             db.session.commit()
         return True
 
+    def validate_worklogs(self):
+        worklogs = self.worklogs
+        target_name = self.target.connector_type.name
+
+        target_connector = ConnectorManager.create_connector(
+            target_name,
+            server=self.target.server,
+            api_token=self.target.api_token,
+            login=self.target.login,
+            password=self.target.password
+        )
+        
+        for worklog in worklogs:

```

Comment:
You shouldn't check worklogs with is_valid=False because they are already invalid :)